content/en/news/releases/1.19.x/announcing-1.19/change-notes/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Istio 1.19.0 Change Notes
+linktitle: 1.19.0
+subtitle: Minor Release
+description: Istio 1.19.0 change notes.
+publishdate: 2023-09-01
+release: 1.19.0
+weight: 20
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+- There are no new deprecations in Istio 1.19.0
+
+## Traffic Management
+
+- **Improved** JWT claim based routing to support using `[]` as a separator for nested claim names.
+  ([Issue #44228](https://github.com/istio/istio/issues/44228))
+
+- **Improved** performance of sidecar injection, in particular with pods with a large number of environment variables.
+
+- **Updated** DNS resolution when using `ServiceEntries` so that DNS for multi-network gateways.
```

Comment:
```suggestion
- **Updated** DNS resolution when using `ServiceEntries` so that DNS for multi-network gateways
```

No period here, next line down is the last part of the sentence